packages/client/hmi-client/src/services/save-asset.ts

```diff
@@ -47,24 +51,27 @@ export async function saveAs(
 		return;
 	}
 
-	const projectId = useProjects().activeProject.value?.id;
-	if (!projectId) {
-		logger.error(`Asset can't be saved since target project doesn't exist.`);
-		return;
-	}
-	await useProjects().addAsset(assetType, response.id, projectId);
+	// TODO: Potentially add a flag in case we don't want to add the asset to the project
+	if (assetType !== AssetType.InterventionPolicy) {
```

Comment:
Isn't this what the function parameter `openOnSave` is suppose to guard against?

Reply:
Resolved in a chat with Cole. There are two things here
- whether we want to save into project-asset, and
- whether we wan to navigate to the asset

So these are handled in a single if-block ... may be clearer if they are independent.

Reply:
I've made saving and navigating into separate if-blocks